src/FocusRingContext.tsx

```diff
@@ -24,6 +24,10 @@ function setActiveRingContextManager(manager: FocusRingContextManager) {
   }
 }
 
+function parseBorderRadius(radius: string) {
```

Comment:
Since this function is called with an optional property chain (`styles[0]?.property`), this could actually be an undefined value, so `radius: string | undefined`. I'd leave it in the long-hand rather than `radius?: string` since the argument is still _required_, it may just be undefined.

Reply:
Good point! Fixed it.